src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -193,19 +191,19 @@ <h2 class="card-pf-title">
                           </div>
                         </div>
                         <div class="list-group-item-text" *ngIf="runtime.description">
```

Comment:
This should extend underneath the dropdown selection, not be inline (or stop) before the dropdown.

The dropdown should be inline with the name of the runtime.